docs/stores/vuex/vuex.md

```diff
@@ -131,17 +416,15 @@ Using store state in a component simply involves returning the state within a co
 Here's the complete example:
 
 <iframe
-     src="https://codesandbox.io/embed/elegant-bogdan-e92qy?fontsize=14&hidenavigation=1&theme=dark&view=editor"
+     src="https://codesandbox.io/embed/simplest-counter-example-e92qy?fontsize=14&hidenavigation=1&module=%2Fsrc%2FApp.vue&theme=dark&view=editor"
      style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
-     title="elegant-bogdan-e92qy"
+     title="SImplest counter example"
```

Comment:
please change to Simplest counter example

Reply:
Done !